libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -8,15 +8,34 @@
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
       </div>
+      <div *ngIf="(b.finished)" class="finished" data-testing="reading-list-finished">
+        <mat-icon class="finished">done</mat-icon>Finished on {{b.finishedDate | date: 'dd/MM/yyyy'}}
+      </div>
     </div>
     <div>
-      <button
-        mat-icon-button
-        color="warn"
-        (click)="removeFromReadingList(b)"
-      >
-        <mat-icon attr.aria-label="Remove {{b.title}} from reading list">remove_circle</mat-icon>
```

Comment:
This syntax (`attr.aria-label="Remove {{b.title}}...`) is not correct. Please take a look here: [https://angular.io/guide/attribute-binding](https://angular.io/guide/attribute-binding)

Reply:
My bad, this one went through on my revision for the first task somehow, fixed now.